recipe/meta.yaml

```diff
@@ -2,8 +2,13 @@ package:
   name: tensorflow
   version: "0.9.0"
 
+source:
+  fn: v0.9.0.tar.gz
+  url: https://github.com/tensorflow/tensorflow/archive/v0.9.0.tar.gz
```

Comment:
Kind of a small thing, but version templating might be a nice thing. Not a blocker by any means. Just mentioning it.